src/velo-fed/BaseFedCCTP.sol

```diff
@@ -166,105 +222,129 @@ contract BaseFedCCTP {
+     */
+    function setMaxSlippageUsdcToDola(
+        uint256 newMaxSlippageBps
+    ) external onlyGov {
         if (newMaxSlippageBps > 10000) revert MaxSlippageTooHigh();
         maxSlippageBpsUsdcToDola = newMaxSlippageBps;
     }
 
     /**
-    @notice Method for `gov` to change `pendingGov` address
-    @dev `pendingGov` will have to call `claimGov` to complete `gov` transfer
-    @param newPendingGov_ Address to be set as `pendingGov`
-    */
-    function setPendingGov(address newPendingGov_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        pendingGov = newPendingGov_;
+     * @notice Method for `gov` to change `pendingGov` address
+     * @dev `pendingGov` will have to call `claimGov` to complete `gov` transfer
+     * @param newPendingGov Address to be set as `pendingGov`
+     */
+    function setPendingGov(address newPendingGov) external onlyGov {
+        pendingGov = newPendingGov;
     }
 
     /**
-    @notice Method for `pendingGov` to claim `gov` role.
-    */
+     * @notice Method for `pendingGov` to claim `gov` role.
+     */
     function claimGov() external {
         if (msg.sender != pendingGov) revert OnlyPendingGov();
         gov = pendingGov;
         pendingGov = address(0);
     }
 
     /**
-    @notice Method for gov to change the chair
-    @param newChair_ Address to be set as chair
-    */
-    function changeChair(address newChair_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        chair = newChair_;
-    }
-
-    /**
-    @notice Method for gov to change the L2 aeroFarmer address
-    @dev aeroFarmer is the L2 address that receives all bridged DOLA from expansion
-    @param newAeroFarmer_ L2 address to be set as aeroFarmer
-    */
-     function changeAeroFarmer(address newAeroFarmer_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        aeroFarmer = newAeroFarmer_;
+     * @notice Method for gov to change the chair
+     * @param newChair Address to be set as chair
+     */
+    function changeChair(address newChair) external onlyGov {
+        chair = newChair;
     }
 
     /**
-    @notice Method for gov to change the curve pool address
-    @param newCurvePool_ Address to be set as curvePool 
-    @param _dolaIndex int of the dola coin index in the new curvepool
-    @param _usdcIndex int of the usdc coin index in the new curvepool
-    */
-     function changeCurvePool(address newCurvePool_, int128 _dolaIndex, int128 _usdcIndex) external {
-        if (msg.sender != gov) revert OnlyGov();
-        dolaIndex = _dolaIndex;
-        usdcIndex = _usdcIndex;
-        curvePool = ICurvePool(newCurvePool_);
+     * @notice Method for gov to change the L2 aeroFarmer address
+     * @dev aeroFarmer is the L2 address that receives all bridged DOLA from expansion
+     * @param newAeroFarmer L2 address to be set as aeroFarmer
+     */
+    function changeAeroFarmer(address newAeroFarmer) external onlyGov {
+        aeroFarmer = newAeroFarmer;
```

Comment:
Worth it to add revert on zero address to avoid any UI accidents.
```suggestion
    function changeAeroFarmer(address newAeroFarmer) external onlyGov {
        if(newAeroFarmer == address(0)) revert ZeroAddressParameter();
        aeroFarmer = newAeroFarmer;
```